cmd/validatemodels/validate_models.go

```diff
@@ -144,7 +144,7 @@ func ValidateAllAuthorizationModels(ctx context.Context, db storage.OpenFGADatas
 					validationResults = append(validationResults, validationResult)
 				}
 
-				continuationTokenModels = string(tokenModels)
+				continuationTokenModels = tokenModels
```

Comment:
Suggestion: same as the other comments, this isn't a continuation token anymore, it's a ulid. Could we rename the variables?